Examples/ExampleMultiProjectIntegration/ExampleMultiProjectIntegration/Views/ExampleApp.swift

```diff
@@ -0,0 +1,71 @@
+// Distributed under the MIT License
+//
+// Permission is hereby granted, free of charge, to any person obtaining a copy
+// of this software and associated documentation files (the "Software"), to deal
+// in the Software without restriction, including without limitation the rights
+// to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+// copies of the Software, and to permit persons to whom the Software is
+// furnished to do so, subject to the following conditions:
+//
+// The above copyright notice and this permission notice shall be included in all
+// copies or substantial portions of the Software.
+//
+// THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+// IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+// FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+// AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+// LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+// OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+// SOFTWARE.
+
+import Combine
+import SafeDI
+import Subproject
+import SwiftUI
+
+// @Instantiable macro marks this type as capable of being instantiated by SafeDI.
+@Instantiable
+@MainActor
+@main
+public struct NotesApp: Instantiable, App {
+    public var body: some Scene {
+        WindowGroup {
+            if let userName = userService.userName {
+                // Returns a new instance of `NoteView`.
+                noteViewBuilder.instantiate(userName)
+            } else {
+                // Returns a new instance of a `NameEntryView`.
+                nameEntryViewBuilder.instantiate()
+            }
+        }
+    }
+
+    // Memberwise initializer to satisfy SafeDI.
+    // `public init()` will be generated for this type because this type is a root.
+    // SafeDI identifies this type as a root because:
+    // 1. It only has `@Instantiated` properties (nothing `@Received` or `@Forwarded`)
+    // 2. No other type has an `@Instantiated` property of this type
+    public init(
```

Comment:
Since this is generated by SafeDI, does it make more sense to keep the code here but comment it out? That makes the code generation more obvious IMO.